app/src/main/res/values/strings.xml

```diff
@@ -167,4 +167,6 @@
     <string name="saving_dialog">Saving..</string>
     <string name="choose_filter">Choose a Filter First</string>
     <string name="none_filter">None</string>
+    <string name="warning">Warning</string>
+    <string name="overwrite_message">File with same name exist, would you like to overwrite?</string>
```

Comment:
Change the message to `There already exists a file with the same name. Do you want to overwrite?`